package/src/contexts/messageInputContext/MessageInputContext.tsx

```diff
@@ -1002,32 +992,59 @@ export const MessageInputProvider = <
   }) => {
     const id = generateRandomId();
     const mimeType = lookup(file.name);
+
+    const blockedFile = getBlockedFileExtensionTypes?.some((fileExtensionType: string) =>
```

Comment:
```suggestion
    const isBlockedFile = getBlockedFileExtensionTypes?.some((fileExtensionType: string) =>
```

`is` makes it clear to the reader that its a boolean

Reply:
✅